api/v1/words/routes.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const Joi = require('joi');
+
+module.exports = [
+  {
+    method: 'GET',
+    path: '/api/v1/words',
+    config: {
+      validate: {
+        query: {
+          wordname: Joi.string(),
+          userid: Joi.string(),
+          tag: Joi.string()
```

Comment:
I made this as tag and not 'tags' as only one search key will be searched/queried at a time.